setuper/additional_projects.py

```diff
@@ -0,0 +1,117 @@
+def projects_mapper(account_name):
+    projects = [
+        {
+            "name": "Planning",
+            "app_id": f"{account_name}.planning",
+            "feature_flags": ["REQUIRE_AUTHENTICATION", "TAKE_GPS_ON_FORM", "", "FORMS_AUTO_UPLOAD", "PLANNING"],
```

Comment:
```suggestion
            "feature_flags": ["REQUIRE_AUTHENTICATION", "TAKE_GPS_ON_FORM", "FORMS_AUTO_UPLOAD", "PLANNING"],
```

I think that the empty perm is a typo?

Reply:
yes, It was a mistake from my side! 
Thanks!